Gopkg.lock

```diff
@@ -86,6 +86,12 @@
   packages = ["."]
   revision = "3955978caca48c1658a4bb7a9c6a0f084e326af3"
 
+[[projects]]
```

Comment:
We have to make sure other project doesn't use this, or use the same version of this package, or at least compatible to this package version.

Reply:
In general, you should not review changes made in `Gopkg.lock` because such file is generated by `dep`.